app/Http/Controllers/Api/V1/CurrikiGo/LmsServicesController.php

```diff
@@ -80,6 +82,7 @@ public function getXAPIFile(Request $request, Activity $activity) {
     }
 
     /**
+     * Canvas Teacher's data.
```

Comment:
Add new line between title and description.

Reply:
Issue still persists.

Reply:
Done!